### DISCUSSION TITLE
Make copilot suggestion take precedence over cmp

### DISCUSSION THREAD

**Discussion by srcrip** [hrsh7th/nvim-cmp#1138] [Q&A]
I am specifically *not* using the cmp-copilot plugin, so I'm not using copilot as a source. I use them separately.

My question is, how can I make it so that when I accept the next cmp suggestion with `tab`, if there is a current pending copilot suggestion, it accepts that *instead*?

**Comment by Shougo** [comment]
Please see this.

https://github.com/github/copilot.vim/blob/release/doc/copilot.txt#L73

**Reply by Shougo** [reply]
Tab is already mapped by copilot.vim.  You need to disable it if you want to use tab key.

**Comment by hrsh7th** [comment]
Please read the nvim-cmp's documents.
https://github.com/hrsh7th/nvim-cmp/blob/main/doc/cmp.txt#L816
